code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -1,6 +1,8 @@
 GLOBAL_LIST_EMPTY(roundstart_races)
 ///List of all roundstart languages by path
 GLOBAL_LIST_EMPTY(roundstart_languages)
+///List of all roundstart languages by path except common
+GLOBAL_LIST_EMPTY(uncommon_roundstart_languages)
```

Comment:
Just a request, can you make this replace ``roundstart_languages`` entirely? It's making billingual's Random option be able to roll common at the moment, and the only time it is used manually filters out common anyways.